.github/CHANGELOG.md

```diff
@@ -77,6 +77,7 @@
 1. [LIGHTS] Movement of landing lights now requires power and position is output into LVAR -  @Maximilian-Reuter
 1. [CDU] Fix auto weight and balance import on INIT B during GSX boarding not using the target values - @Maximilian-Reuter
 2. [FAC] Improve sideslip estimation - @lukecologne (luke)
+1. [EFB] Added missing localization for SimBridge related settings in SimOptions page - @implasmatbh (Plasma)
```

Comment:
A changelog is probably not strictly necessary as there's no user-visible change. PR looks good otherwise :+1:.

Reply:
I see. Should I then remove it from the changelog?
Also I need to get a build of this plane to make sure that the changes actually work but GitHub is skipping the build test. Am I missing something to get this to work?